server/src/test/java/org/opensearch/action/admin/cluster/health/ClusterHealthResponsesTests.java

```diff
@@ -112,9 +112,9 @@ public void testClusterHealth() throws IOException {
         assertThat(clusterHealth.getActiveShardsPercent(), is(allOf(greaterThanOrEqualTo(0.0), lessThanOrEqualTo(100.0))));
     }
 
-    public void testClusterHealthVerifyMasterNodeDiscovery() throws IOException {
+    public void testClusterHealthVerifyClusterManagerNodeDiscovery() throws IOException {
         DiscoveryNode localNode = new DiscoveryNode("node", OpenSearchTestCase.buildNewFakeTransportAddress(), Version.CURRENT);
-        // set the node information to verify master_node discovery in ClusterHealthResponse
+        // set the node information to verify cluster_manager_node discovery in ClusterHealthResponse
         ClusterState clusterState = ClusterState.builder(ClusterName.CLUSTER_NAME_SETTING.getDefault(Settings.EMPTY))
             .nodes(DiscoveryNodes.builder().add(localNode).localNodeId(localNode.getId()).masterNodeId(localNode.getId()))
```

Comment:
`masterNodeId` -> `clusterManagerNodeId`?

Reply:
I will rename `masterNodeId` in the next phase, which is mentioned in issue https://github.com/opensearch-project/OpenSearch/issues/1684. It's a public method, changing the name will be a breaking change, so I will do it in the next major version. 😄